src/main/java/com/fp3/haras/view/screens/auth/LoginScreen.java

```diff
@@ -0,0 +1,258 @@
+                .addContainerGap())
+        );
+        panelBackLayout.setVerticalGroup(
+            panelBackLayout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
+            .addGroup(panelBackLayout.createSequentialGroup()
+                .addGap(13, 13, 13)
+                .addComponent(lblTitle)
+                .addGap(18, 18, 18)
+                .addComponent(panelCenter, javax.swing.GroupLayout.PREFERRED_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.PREFERRED_SIZE)
+                .addPreferredGap(javax.swing.LayoutStyle.ComponentPlacement.RELATED, 54, Short.MAX_VALUE)
+                .addComponent(jLabel1)
+                .addGap(0, 0, 0)
+                .addComponent(jLabel2)
+                .addContainerGap())
+        );
+
+        javax.swing.GroupLayout layout = new javax.swing.GroupLayout(getContentPane());
+        getContentPane().setLayout(layout);
+        layout.setHorizontalGroup(
+            layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
+            .addComponent(panelBack, javax.swing.GroupLayout.PREFERRED_SIZE, 674, javax.swing.GroupLayout.PREFERRED_SIZE)
+        );
+        layout.setVerticalGroup(
+            layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
+            .addComponent(panelBack, javax.swing.GroupLayout.DEFAULT_SIZE, 580, Short.MAX_VALUE)
+        );
+
+        pack();
+        setLocationRelativeTo(null);
+    }// </editor-fold>//GEN-END:initComponents
+
+    private String getPassword() {
+        return String.valueOf(txtPass.getPassword());
+    }
+    
+    private void btnEnterMouseEntered(java.awt.event.MouseEvent evt) {//GEN-FIRST:event_btnEnterMouseEntered
+        btnEnter.setBackground(Colors.ACCENTBLUE);
+    }//GEN-LAST:event_btnEnterMouseEntered
+
+    private void txtUserFocusGained(java.awt.event.FocusEvent evt) {//GEN-FIRST:event_txtUserFocusGained
+        txtUser.setText(null);
+    }//GEN-LAST:event_txtUserFocusGained
+
+    private void txtPassFocusGained(java.awt.event.FocusEvent evt) {//GEN-FIRST:event_txtPassFocusGained
+        txtPass.setText(null);
+    }//GEN-LAST:event_txtPassFocusGained
+
+    private void txtPassFocusLost(java.awt.event.FocusEvent evt) {//GEN-FIRST:event_txtPassFocusLost
+        if (getPassword().equals(""))
+            txtPass.setText("Senha");
+    }//GEN-LAST:event_txtPassFocusLost
+
+    private void txtUserFocusLost(java.awt.event.FocusEvent evt) {//GEN-FIRST:event_txtUserFocusLost
+        if (txtUser.getText().equals(""))
+            txtUser.setText("Administrador");
+    }//GEN-LAST:event_txtUserFocusLost
+
+    private void btnEnterActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_btnEnterActionPerformed
+        if (!txtUser.getText().equals("") && !txtUser.getText().equals("Administrador")
+        && !getPassword().equals("") && !getPassword().equals("Senha")) {
```

Comment:
I think this condition could be simplified to something like `textUser.getText() && !txtUser.getText().equals("Administrador") && getPassword()  && !getPassword().equals("Senha")`. Test this and change it if works